homeassistant/components/usb/__init__.py

```diff
@@ -48,28 +50,19 @@ def __init__(
         self.flow_dispatcher = flow_dispatcher
         self.usb = usb
         self.seen: set[tuple[str, ...]] = set()
+        self.observer_active = False
 
     async def async_setup(self) -> None:
         """Set up USB Discovery."""
-        if not await self._async_start_monitor():
-            await self._async_start_scanner()
+        await self._async_start_monitor()
         self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STARTED, self.async_start)
 
     async def async_start(self, event: Event) -> None:
         """Start USB Discovery and run a manual scan."""
         self.flow_dispatcher.async_start()
         await self.hass.async_add_executor_job(self.scan_serial)
 
-    async def _async_start_scanner(self) -> None:
-        """Perodic scan with pyserial when the observer is not available."""
-        stop_track = async_track_time_interval(
-            self.hass, lambda now: self.scan_serial(), SCAN_INTERVAL
-        )
-        self.hass.bus.async_listen_once(
-            EVENT_HOMEASSISTANT_STOP, callback(lambda event: stop_track())
-        )
-
-    async def _async_start_monitor(self) -> bool:
+    async def _async_start_monitor(self) -> None:
         """Start monitoring hardware with pyudev."""
         if not sys.platform.startswith("linux"):
             return False
```

Comment:
There's one more below too.
```suggestion
            return
```

Reply:
Thanks